.gitignore

```diff
@@ -71,3 +71,7 @@ Guardfile
 .vagrant/
 tags
 .byebug_history
+
+# testing cache
+/features/support/fixtures/req_cache
+/features/support/fixtures/cassettes
```

Comment:
I am.  If you add those lines to gitignore then our acceptance tests will not be properly sandboxed by vcr and puffing billy:

* https://github.com/vcr/vcr
* https://github.com/oesmith/puffing-billy

We can't merge in with those lines present.

If you are getting stray files in those directories it means that you are experiencing network "leaks" when you are running the acceptance tests.

Ideally one would be investigating where those leaks are coming from, although that can be a complex topic.  It would be relatively easy for me to show you in a pairing session, or you can research the github links above.